### DISCUSSION TITLE
standaard-2-de-onderwijsleeromgeving

### DISCUSSION THREAD

**Discussion by fjkresin** [fdnd/docs.fdnd.nl#277] [Reflectie]
Voor een paragraaf met het kopje 'blijvende zorg' is het antwoord op 'hoe gaan we daarmee om' wat kort; ik zou dat nog wat verder toelichten.

**Comment by ju5tu5** [comment]
Yes! Is opgenomen maar op aanraden van O&O expliciet kort gehouden, de langere uitleg staat bij _Aansluiten op de actualiteit_
